backend/stakepoold/userdata/userdata.go

```diff
@@ -16,7 +16,7 @@ type DBConfig struct {
 
 // UserData stores the current snapshot of the user voting config.
 type UserData struct {
-	sync.RWMutex
+	mtx              sync.RWMutex
```

Comment:
Don't use mtx; make it nameless so that the data structure itself can be locked.  That is the idomatic way.